client/llb/fileop.go

```diff
@@ -336,6 +347,51 @@ func Mkfile(p string, m os.FileMode, dt []byte, opts ...MkfileOption) *FileActio
 	}
 }
 
+type SymlinkInfo struct {
+	ChownOpt    *ChownOpt
```

Comment:
`Mode` as well

Reply:
According to the man pages, symlink permissions are irrelevant. Is the mode for the SUID, SGID, and sticky bits? What effect do those have on symlinks?